tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -33,6 +33,8 @@ constexpr size_t MAX_MEMORY = 64e+9;
 
 using namespace glow;
 
+extern llvm::cl::opt<std::string> loadDeviceConfigsFileOpt;
```

Comment:
May be this declaration should be in HostManager.h and then you wouldn't need it here?